test/Common/utils.test.ts

```diff
@@ -162,13 +163,14 @@ describe('utils', function (): void {
       classicAddress!.address,
       'rU6K7V3Po4snVhBBaU29sesqs2qTQJWDw1',
     )
-    assert.strictEqual(classicAddress!.tag, 12345)
+    assert.strictEqual(classicAddress!.tag, tag)
```

Comment:
Added a variable for this to avoid the `no-magic-numbers` rule. This seems reasonable because the `GIVEN` comment includes the tag